### ISSUE TITLE
Can't clone on Windows because it has a Question mark on title number 18. 

### ISSUE THREAD

**Issue by drewanfrasier** [scrimba/learn-javascript#11]
https://github.com/scrimba/learn-javascript/tree/main/1.%20Build%20a%20Passenger%20Counter%20App/18.%20What%20is%20a%20string%3F

**Comment by deep-woods** [comment]
Same issue here. 

``` git clone https://github.com/scrimba/learn-javascript.git

Cloning into 'learn-javascript'...
remote: Enumerating objects: 590, done.
remote: Counting objects: 100% (192/192), done.
remote: Compressing objects: 100% (134/134), done.
remote: Total 590 (delta 88), reused 58 (delta 58), pack-reused 398Receiving objects:  90% (531/590), 2.71 MiB | 5.40 MiReceiving objects:  91% (537/590), 2.71 MiB | 5.40 MiB/s
Receiving objects: 100% (590/590), 4.27 MiB | 5.85 MiB/s, done.
Resolving deltas: 100% (257/257), done.
error: invalid path '1. Build a Passenger Counter App/18. What is a string?/index.css'    👈👈👈 --- PROBLEM
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

**Comment by Lovelyfin00** [comment]
Same issue here

Cloning into 'learn-javascript'...
remote: Enumerating objects: 590, done.
remote: Counting objects: 100% (192/192), done.
remote: Compressing objects: 100% (134/134), done.
remote: Total 590 (delta 88), reused 58 (delta 58), pack-reused 398
Receiving objects: 100% (590/590), 4.27 MiB | 1.33 MiB/s, done.
Resolving deltas: 100% (257/257), done.
error: invalid path '1. Build a Passenger Counter App/18. What is a string?/index.css'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

**Comment by jeremyers1** [comment]
Still an issue. This will not clone on windows VS Code with Bash. 

I get the exact same errors as above.

**Comment by sarantsatsral-qy** [comment]
Same issue here. 

![image](https://user-images.githubusercontent.com/30141705/223989490-7d82f9f4-8247-4b7f-b7b5-b0a97f72ed6b.png)

**Comment by soham124** [comment]
Still reproduces today (August 2026) on Windows 11 / git 2.x — `git clone` fails at checkout with:

```
fatal: cannot create directory at '1. Build a Passenger Counter App/18. What is a string?': Invalid argument
warning: Clone succeeded, but checkout failed.
```

**Scope:** 9 directories, 31 of the repo's 481 files. The offending characters are `?`, `:`, `<`, `>`, `*` — none of which are legal in NTFS filenames, so this is not fixable client-side. Downloading the ZIP fails the same way, since it carries the same directory names.

**There are already four open PRs that fix this, all currently `MERGEABLE` with no conflicts:**

- #15 (jeremyers1, Jul 2022) — 31 files, pure directory renames, `+0/-0`
- #29 (joce, Aug 2023) — 31 files, pure directory renames, `+0/-0`
- #26 (Alachi24, Jun 2023) — partial, covers only lesson 18
- #41 (manojr1r, Mar 2026) — full fix, but also deletes 538 lines across 62 files

I verified #15 and #29 locally: both produce all 481 files with zero Windows-illegal paths, renaming only directories and leaving every file's contents byte-identical. Either one resolves this issue as-is — **#29 or #15 are the minimal changes** if you'd prefer the smallest diff.

Related: #1 (closed 2021, unfixed), #28.

Merging any one of these would unblock every Windows student who hits this. It has been open since June 2021 across three separate issue reports.
